### PR TITLE
Create generic attention variant

### PR DESCRIPTION
**Pull request by richford**
This PR is WIP but starts implementing a generic attention task. We can use the auto-generated test URL to test out the UX

### CODE REVIEW

**Comment by richford** [comment]
Be sure to append `?dots=true` to the generated URL

**Comment by ksmontville** [comment]
Task linked: [CU-8686pf298 Create roar-mep game new version](https://app.clickup.com/t/8686pf298)

**Comment by richford** [comment]
Remaining tasks:
- [ ] Don't change opacity on the underlining
- [ ] Maha, upload new introduction videos

**Comment by richford** [comment]
@Ram-Maha , I noticed that you referenced video files in the src/assets/video/es/generic folder but did not add any of those files. Did you perhaps forget to `git add` those files?

File: src/preload.js [hunk 1]
```diff
@@ -53,6 +54,29 @@ import preCuePostBlock2 from "./assets/video/en/precue/postBlock2.mp4";
...
+import genericIntro1 from "./assets/video/es/generic/intro1Generic.mp4";
+import genericIntro2 from "./assets/video/es/generic/intro2Generic.mp4";
+import genericIntro3 from "./assets/video/es/generic/intro3Generic.mp4";
+import genericIntro4 from "./assets/video/es/generic/intro4Generic.mp4";
+import genericPostPractice from "./assets/video/es/generic/postPracticeGeneric.mp4";
+import genericPostBlock1 from "./assets/video/es/generic/postBlock1Generic.mp4";
+import genericPostBlock2 from "./assets/video/es/generic/postBlock2Generic.mp4";
+import genericPostBlock3 from "./assets/video/es/generic/postBlock3Generic.mp4";
+import genericPostBlock4 from "./assets/video/es/generic/postBlock4Generic.mp4";
+import genericEnd from "./assets/video/es/generic/endGeneric.mp4";
```
Comment (richford) [inline src/preload.js:79]: The imported Spanish videos will need to have slightly different variable names than the English ones. Suggestion below

```suggestion
import genericIntro1Es from "./assets/video/es/generic/intro1Generic.mp4";
import genericIntro2Es from "./assets/video/es/generic/intro2Generic.mp4";
import genericIntro3Es from "./assets/video/es/generic/intro3Generic.mp4";
import genericIntro4Es from "./assets/video/es/generic/intro4Generic.mp4";
import genericPostPracticeEs from "./assets/video/es/generic/postPracticeGeneric.mp4";
import genericPostBlock1Es from "./assets/video/es/generic/postBlock1Generic.mp4";
import genericPostBlock2Es from "./assets/video/es/generic/postBlock2Generic.mp4";
import genericPostBlock3Es from "./assets/video/es/generic/postBlock3Generic.mp4";
import genericPostBlock4Es from "./assets/video/es/generic/postBlock4Generic.mp4";
import genericEndEs from "./assets/video/es/generic/endGeneric.mp4";
```

File: src/preload.js [hunk 2]
```diff
@@ -89,7 +113,25 @@ const preload_character_trials = {
 
 let videoBlocks;
 if (config.language === "en") {
-  if (config.precue) {
+  if (config.dots) {
+    // TODO (Maha): Any video assets that were imported above should also be
+    // included in the array below, by referencing the variable name that you
+    // used in the import statement.
```
Comment (richford) [inline src/preload.js:119]:
```suggestion
```

File: src/videos.js [hunk 3]
```diff
@@ -35,10 +37,124 @@ const clickStarTrial = {
...
+  }
+};
+
 let videoTrials;
 
 /* define instructions trial */
-if (config.precue) {
+if (config.dots) {
+  // TODO (Maha): Here is where we define the video trials for the generic
+  // attention variant.
```
Comment (richford) [inline src/videos.js:82]:
```suggestion
```

File: src/videos.js [hunk 4]
```diff
@@ -35,10 +37,124 @@ const clickStarTrial = {
...
+  const introTrial1 = [{
+    // Note (Maha): The key name in the ``videoContent`` object matches the
+    // filename of the video asset (without the .mp4 extension).
```
Comment (richford) [inline src/videos.js:85]:
```suggestion
```

File: src/videos.js [hunk 5]
```diff
@@ -35,10 +37,124 @@ const clickStarTrial = {
...
+    stimulus: [videoContent.intro1Generic],
+    ...kwargs,
+  }];
+
+  const introTrial2 = [{
+    stimulus: [videoContent.intro2Generic],
+    ...kwargs,
+    // TODO (Maha): The parameter below must be added in order to require a
+    // click on the left side of the screen after the video ends.
```
Comment (richford) [inline src/videos.js:94]:
```suggestion
```

File: src/videos.js [hunk 6]
```diff
@@ -35,10 +37,124 @@ const clickStarTrial = {
...
+    trial_ends_after_video: false,
+    choices: ["ArrowLeft"],
+    on_load: loadPracticeDivs,
+  }];
+
+  const introTrial3 = [{
+    stimulus: [videoContent.intro3Generic],
+    ...kwargs,
+    // TODO (Maha): The parameter below must be added in order to require a
+    // click on the right side of the screen after the video ends.
```
Comment (richford) [inline src/videos.js:104]:
```suggestion
```

File: src/preload.js (hunk 2 above)
Comment (richford) [reply src/preload.js:119]: Sure! That seems like a good approach to me.

**Pull request merged**